src/App.tsx

```diff
@@ -1,14 +1,32 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions as todoActions } from './features/todos';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const isSelected = currentTodo !== null;
+
+  useEffect(() => {
+    setIsLoading(true);
+    getTodos()
+      .then(todosFromServer => dispatch(todoActions.setTodos(todosFromServer)))
+      .catch(() => {
+        throw new Error('Can`t find todos');
+      })
```

Comment:
No need to throw an error here, you should handle that error and show it to user on UI